### PR TITLE
Apply styles also in the #wrapper to make those work with style books

### PR DESCRIPTION
**Pull request by victorg1991**
In order to avoid admin controls to take the stylebook styles, we decided to apply those styles in the #wrapper element of a page, which resulted in some tokens not working.

The specific tokens that don't work are the ones defined by the theme in the body:

```
body {
    background-color: var(--body-bg);
    color: var(--body-color);
    font-family: var(--font-family-base);
    font-size: var(--font-size-base);
}
```

After a discussion in https://liferay.slack.com/archives/CNBG06JS3/p1598612754140300 we've concluded that a viable solution will be to apply those styles also in the wrapper element

```
#wrapper {
    background-color: var(--body-bg);
    color: var(--body-color);
    font-family: var(--font-family-base);
    font-size: var(--font-size-base);
}
```

### Steps to try the PR

Steps to reproduce

- Go to Style Books
- Create a new stylebook
- Go to typography category
- Update font family base value and set 'Courier New'

### CODE REVIEW

File: modules/apps/frontend-theme/frontend-theme-classic/src/css/custom_properties/_custom_properties_set.scss [hunk 1]
```diff
@@ -48,6 +48,13 @@ body {
 	font-size: var(--font-size-base);
 }
 
+#wrapper {
```
Comment (jbalsas) [inline modules/apps/frontend-theme/frontend-theme-classic/src/css/custom_properties/_custom_properties_set.scss:51]: can't you just do `body, #wrapper`?

Comment (victorg1991) [reply modules/apps/frontend-theme/frontend-theme-classic/src/css/custom_properties/_custom_properties_set.scss:45]: fixed!

**Comment by victorg1991** [comment]
ci:test:sf

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by victorg1991** [comment]
ci:test:relevant

**Comment by pat270** [comment]
LGTM!

**Comment by jbalsas** [comment]
Sending this forward before @pat270 changes his mind!

**Comment by jbalsas** [comment]
ci:forward

**Pull request closed without merging**